diff --git i/build-unsigned.sh w/build-unsigned.sh
index b996a66..41bbdf1 100755
--- i/build-unsigned.sh
+++ w/build-unsigned.sh
@@ -65,6 +65,7 @@ for pair in "${TARGETS[@]}"; do
     -derivedDataPath "$DERIVED" \
     -destination 'generic/platform=macOS' \
     CODE_SIGNING_ALLOWED=NO \
+    MACOSX_DEPLOYMENT_TARGET=12.0 \
     build >/dev/null
 
   app="$PRODUCTS/$scheme.app"
diff --git c/build-signed.sh w/build-signed.sh
index 3781b16..fd42050 100755
--- c/build-signed.sh
+++ w/build-signed.sh
@@ -123,6 +123,7 @@ for pair in "${TARGETS[@]}"; do
     -derivedDataPath "$DERIVED" \
     -destination 'generic/platform=macOS' \
     CODE_SIGNING_ALLOWED=NO \
+    MACOSX_DEPLOYMENT_TARGET=12.0 \
     build >/dev/null
 
   app="$PRODUCTS/$scheme.app"
